packages/@aws-cdk/aws-ecs-patterns/README.md

```diff
@@ -296,6 +296,8 @@ const ecsScheduledTask = new ScheduledEc2Task(stack, 'ScheduledTask', {
 });
 ```
 
+This task will use the `LATEST` platform version.
```

Comment:
curious about this note: is the recommendation to guide users towards setting a platform version? why not set it in the example itself to use a specific platform version?

might also be helpful to include a link to documentation around setting a platform version

Reply:
I thought the example code in the README should be simple.
It is simpler to omit all but the required parameters.